interface/ceed-qfunction.c

```diff
@@ -81,6 +81,32 @@ int CeedQFunctionCreateInterior(Ceed ceed, CeedInt vlength,
   return 0;
 }
 
+/**
+  @brief Create a CeedQFunction from a named CeedQFunction in the gallery
+
+  @param ceed       A Ceed object where the CeedQFunction will be created
+  @param name       Name of the QFuction in the gallery
+  @param[out] qf    Address of the variable where the newly created
+                     CeedQFunction will be stored
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Basic
+**/
+int CeedQFunctionCreateFromGallery(Ceed ceed, const char *name,
```

Comment:
Intentionally left just as comments for now, I see this function as needing to 
1) find the requested qfunction (folder search or registered functions search)
2) set up the requested qfunction (see above about the fields, etc)